ceph-windows-pull-requests/config/definitions/ceph-windows-pull-requests.yml

```diff
@@ -35,7 +35,7 @@
           org-list:
             - ceph
           white-list-target-branches:
```

Comment:
😦

Reply:
The GHPRB plugin still has 'whitelist' hardcoded.  https://github.com/jenkinsci/ghprb-plugin/search?q=whitelist

As does JJB: https://opendev.org/jjb/jenkins-job-builder/src/branch/master/jenkins_jobs/modules/triggers.py#L1235